### PR TITLE
Adding Alan and Grace origin story

### PR DESCRIPTION
**Pull request by cchesser**
Adding the Alan and Grace mascot story. This has a target publish date for 11/23.

### CODE REVIEW

**Comment by kpshek** [comment]
@sambao21 - fyi, in the future you also need to deploy the changes after merging the PR with `rake generate` followed by `rake deploy`. See [this for more](https://github.com/cerner/cerner.github.io/blob/source/UPDATING.markdown#deploying-changes)

Deployed via 1d52fc25

**Comment by sambao21** [comment]
Yeah, I was aware of that and was having issues getting libv8 to compile on my new Mac, and thus still working through that. I had octopress working on my previous device, but fighting el Capitan right now. Sorry for not getting it working. Thanks for stepping in.

**Comment by kpshek** [comment]
Ah, no problem. 😄

**Comment by cchesser** [comment]
Thanks @sambao21 + @kpshek!

File: source/_posts/2016-11-23-alan-and-grace-an-origin-story.markdown [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+{% img right /assets/2016-11-23-alan-and-grace-an-origin-story/grace-hopper-2.jpg Grace Hopper %}
+
+<sub>Sources: <a href="http://www.womenatworkmuseum.org/envision-engineering.html">womenatworkmuseum.org</a>, <a href="https://en.wikipedia.org/wiki/Grace_Hopper">wikipedia.org</a></sub>
+</div>
+
+Alan and Grace represent a beacon of inspiration, providing icons that
+encapsulate and promote a culture of innovation, forward thinking, and just
+plain fun. The creation of mascots Alan and Grace has been a big win for the
+passionate Cerner engineers dedicated to fostering Cerner’s Engineering culture.
+Alan and Grace represent a beacon of inspiration, providing icons that
```
Comment (sambao21) [inline source/_posts/2016-11-23-alan-and-grace-an-origin-story.markdown]: Just noticed that this sentence is a duplicate of the one at the beginning of the paragraph. Is the sentence right before this how it should end? Or was there suppose to be some other sentence here for the ending?

Comment (kpshek) [inline source/_posts/2016-11-23-alan-and-grace-an-origin-story.markdown]: I just removed that duplicate sentence typo in 6053ed6 and deployed it in 5fcd0b

**Pull request merged**